tests/common/mod.rs

```diff
@@ -0,0 +1,85 @@
+use policy_server::config::{Config, Policy, PolicyMode};
```

Comment:
we need the old mod.rs here since otherwise `common.rs` will be interpreted as a test by cargo. See: https://doc.rust-lang.org/book/ch11-03-test-organization.html#submodules-in-integration-tests